services/core/events/blockchain.js

```diff
@@ -35,6 +35,10 @@ module.exports = [
 			coreSocket.socket.on('blocks/change', async data => {
 				logger.debug('Returning block list to the socket.io client...');
 				const restData = await core.getBlocks({ blockId: data.id });
+				core.setLastBlock(restData.data[0]);
+				core.preloadBlocks(202).then(() => {
+					core.removeOrphanedBlocks(202);
```

Comment:
Consider referring to the `config.cacheNumOfBlocks` property instead of hardcoding.